### PR TITLE
feat: add badge for first trait

### PR DESCRIPTION
**Pull request by tcheee**

### CODE REVIEW

File: src/components/ProfilePage/QuestCardDetailled/QuestCardDetailled.tsx [hunk 1]
```diff
@@ -104,22 +111,29 @@ export const QuestCardDetailled = ({
...
-              height={288}
-              style={{
-                borderTopLeftRadius: '8px',
-                borderTopRightRadius: '8px',
-              }}
-            />
+            <>
+              <BadgeRelativeBox>
+                {isTraitsGarded ? <TraitsBox trait={'perp_oors'} /> : undefined}
+              </BadgeRelativeBox>
```
Comment (admsk2) [inline src/components/ProfilePage/QuestCardDetailled/QuestCardDetailled.tsx:124]: How about:
```
{isTraitsGarded &&
   <BadgeRelativeBox>
     <TraitsBox trait={'perp_oors'} />
   </BadgeRelativeBox>
}
```

Comment (tcheee) [reply src/components/ProfilePage/QuestCardDetailled/QuestCardDetailled.tsx:124]: yes

**Review by admsk2 — APPROVED** [review]

**Review by dennyscode — CHANGES_REQUESTED** [review]
Nice!!! Working fine, but only one thing I noticed - maybe we can adjust the zIndex?

image

**Review by oktapodia — APPROVED** [review]
LGTM

**Review by dennyscode — APPROVED** [review]

**Pull request merged**